### ISSUE TITLE
Should this repository have continuous integration on GitHub?

### ISSUE THREAD

**Issue by ProfJanetDavis** [glam-lab/degender-the-web#23]
I would welcome advice/assistance from someone who has prior experience with continuous integration for JavaScript on GitHub. May depend on #2, automated testing.

**Comment by ProfJanetDavis** [comment]
On hold while sorting out #2

**Comment by ProfJanetDavis** [comment]
Significant progress has been made on clone-side CI. Is this still needed?
